src/tools/rbd/action/Resize.cc

```diff
@@ -64,13 +64,6 @@ int execute(const po::variables_map &vm,
     return r;
   }
 
-  librbd::image_info_t info;
-  r = image.stat(info, sizeof(info));
-  if (r < 0) {
-    std::cerr << "rbd: resize error: " << cpp_strerror(r) << std::endl;
-    return r;
-  }
-
```

Comment:
Might be nicer to keep this code and add so that it doesn't attempt to resize if it cannot shrink:

```
  if (info.size > size && !vm["allow-shrink"].as<bool>()) {
    r = -EINVAL;
  } else {
    r = do_resize(....);
  }
```

Reply:
thanks